plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/properties/ObjectPropertyDescriptor.java

```diff
@@ -280,7 +284,12 @@ public boolean hasFeature(@NotNull String feature) {
             case DBConstants.PROP_FEATURE_PASSWORD:
                 return this.isPassword();
         }
-        return false;
+
+        if (ArrayUtils.isEmpty(propInfo.features())) {
```

Comment:
such  check is already exist in ArrayUtils.contains

Reply:
addressed